paket.dependencies

```diff
@@ -14,19 +14,17 @@ source https://api.nuget.org/v3/index.json
 #
 
 nuget NuGet.CommandLine
-
-nuget Microsoft.CodeAnalysis.Analyzers 1.1
 nuget Microsoft.CodeAnalysis 2.0
 
 group Analyzers
 	source https://api.nuget.org/v3/index.json
 	nuget StyleCop.Analyzers
 	nuget Microsoft.CodeAnalysis.FxCopAnalyzers
+	nuget Microsoft.CodeAnalysis.Analyzers 1.1
```

Comment:
Can we not use 2.x yet?

Reply:
I was simply moving the reference and retained the existing 'version pinning'. IIRC, you pinned it because we had issues with 2.6. Might be worth revisiting in a follow-up change.

Reply:
👍